geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped goal
---
bool success
geometry_msgs/PoseStamped[] path
geometry_msgs/PoseStamped[] inters